openmdao/core/system.py

```diff
@@ -2445,13 +2445,36 @@ def _set_solver_print(self, level=2, depth=1e99, type_='all'):
             if current_depth >= depth:
                 continue
 
-            subsys._set_solver_print(level=level, depth=depth - current_depth, type_=type_)
+            subsys.set_solver_print(level=level, depth=depth - current_depth, type_=type_)
 
             if subsys._linear_solver is not None and type_ != 'NL':
                 subsys._linear_solver._set_solver_print(level=level, type_=type_)
             if subsys.nonlinear_solver is not None and type_ != 'LN':
                 subsys.nonlinear_solver._set_solver_print(level=level, type_=type_)
 
+    def _set_solver_print(self, level=2, depth=1e99, type_='all'):
```

Comment:
We shouldn't have to deprecate an underscored function, no?
Is this just because it has become  a _de facto_ API function?

Reply:
I agree, though if you feel you do have to deprecate it, then add a test too.

Reply:
I'm OK with the standard of not deprecating non-API functions, I'll remove the deprecation.